config/default.yml

```diff
@@ -783,6 +783,11 @@ PrePush:
     description: 'Running RSpec test suite'
     required_executable: 'rspec'
 
+  Minitest:
+    enabled: false
+    description: 'Running Minitest test suite'
+    command: ['bundle', 'exec', 'rake', 'test']
```

Comment:
There's nothing about this command that is specific to the `minitest` framework&mdash;it's executing a Rake task that could be executing any code.